servicetemplates/example.org.tosca.servicetemplates/DataPipelineExample/ServiceTemplate.tosca

```diff
@@ -0,0 +1,110 @@
+tosca_definitions_version: tosca_simple_yaml_1_3
```

Comment:
don't you want to change the namespace to `radon.blueprints.examples` or something more `radon` sepcific?

Reply:
Good idea, changed it into radon.blueprints.examples

Reply:
Also pushed changes after POST touch